agent/engine/engine_unix_integ_test.go

```diff
@@ -370,14 +383,15 @@ func TestDynamicPortForward(t *testing.T) {
 	require.Equal(t, event.(api.ContainerStateChange).Status, apicontainerstatus.ContainerRunning, "Expected container to be RUNNING")
 
 	portBindings := event.(api.ContainerStateChange).PortBindings
+	validPortBindings := getValidPortBinding(portBindings)
```

Comment:
discussed offline, we will add a comment to mention why we are filtering it out in the test code

Reply:
Will add comments to explain the fix in next revision. Thanks!